src/main/java/frc/robot/subsystems/Intake/IntakeConstants.java

```diff
@@ -34,8 +33,8 @@ public enum IntakeState {
 
         final double targetMotorOutput;
 
-        IntakeState(double setIntakeState) {
-            this.targetMotorOutput = setIntakeState;
+        IntakeState(double targetIntakeOutput) {
+            this.targetMotorOutput = targetIntakeOutput;
```

Comment:
Just output. Intake is obvious because it's in the intake and this isn't necacerily only going to be used for targetOutput. The states represent different states for the subsystem.